src/main/java/net/sf/jabref/gui/util/GUIUtil.java

```diff
@@ -0,0 +1,39 @@
+package net.sf.jabref.gui.util;
+
+import java.awt.FontMetrics;
+
+import javax.swing.JTable;
+import javax.swing.JTree;
+
+public final class GUIUtil {
+
+    private GUIUtil() {
+        // Private constructor as all methods are static
+    }
+
+    /**
+     * Update table row height to the best possible considering font size changes on the current platform
+     *
+     * http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=4760081
+     *
+     * @param table
+     */
+    public static void correctRowHeight(JTable table) {
```

Comment:
No. JComponent doesn't have a setRowHeight method (I checked it before writing the code) and I do not think it is possible to cast from `JTree` to `JTable` or the opposite.